package com.esgi.nova.application.uses_cases.difficulties.exceptions

class DifficultyNotFoundByIdException : Throwable() {

}
